### PR TITLE
Add autocomplete for `wxf`

### PR DESCRIPTION
**Pull request by Svenito**
for #4751

### CODE REVIEW

**Comment by ret2libc** [comment]
Probably I'm not understand what you want to achieve with this patch :( why would you want to autocomplete the configuration variables when you use `w` and `wx`? Could you provide here an example of the use case of this patch?

**Comment by Svenito** [comment]
Maybe I misunderstood then or edited the wrong file. Latter most likely. Trying to add wxf to the list of commands when typing `w<tab>`. Still figuring out lots on radare

**Comment by ret2libc** [comment]
Ok don't worry :) If you just want to add 'wxf' to the list of commands that are autocompleted, the first line of your patch is enough. The rest should be to autocomplete the arguments of a command. Seems like you copied the autocompletion of the `e` command, that is indeed used to change the variables. Not needed though in this case

File: libr/core/core.c [hunk 1]
```diff
@@ -446,7 +446,7 @@ static const char *radare_argv[] = {
 	"b", "bf", "b?",
 	"/", "//", "/a", "/c", "/m", "/x", "/v", "/v2", "/v4", "/v8", "/r"
 	"y", "yy", "y?",
-	"wx", "ww", "w?",
+	"wx", "ww", "w?", "wxf",
```
Comment (ret2libc) [inline libr/core/core.c]: this line

**Comment by Svenito** [comment]
Ok thanks. I tried just the first line and had no luck. Might have been something else though. I'll double check tomorrow.

**Comment by ret2libc** [comment]
Oh that's weird. I tried that to and when i pressed "w<tab>" i could see "wxf" on the list of possible completions :(

**Comment by Svenito** [comment]
I must have goofed somewhere. Remove the config var block and squashed commits

**Comment by ret2libc** [comment]
Thanks a lot!

**Pull request closed without merging**